src/styles/base/animations.less

```diff
@@ -0,0 +1,265 @@
+// Animations
```

Comment:
For now I have included all of the `base/` stylesheets, but ultimately will remove what is not used.